uniffi_bindgen/src/bindings/kotlin/templates/CallbackInterfaceTemplate.kt

```diff
@@ -1,10 +1,12 @@
-{% import "macros.kt" as kt %}
-{%- let cbi = self.inner() %}
+{%- let cbi = ci.get_callback_interface_definition(name).unwrap() %}
 {%- let type_name = cbi|type_name %}
-{%- let canonical_type_name = cbi|canonical_name %}
-{%- let ffi_converter = cbi|ffi_converter_name %}
 {%- let foreign_callback = format!("ForeignCallback{}", canonical_type_name) %}
 
+{% if self.include_once_check("CallbackInterfaceRuntime.kt") %}{% include "CallbackInterfaceRuntime.kt" %}{% endif %}
+{{- self.add_import("java.util.concurrent.atomic.AtomicLong") }}
+{{- self.add_import("java.util.concurrent.locks.ReentrantLock") }}
+{{- self.add_import("kotlin.concurrent.withLock") }}
```

Comment:
Can you explain the fragile part more?

The `include_once_check` seems a bit fragile to me, since you need to make sure the strings stay in sync.  Is that the issue or is there more?